api/wallet.go

```diff
@@ -196,18 +196,28 @@ func (srv *Server) walletEncryptHandlerPOST(w http.ResponseWriter, req *http.Req
 	}
 	seedStr, err := modules.SeedToString(seed, dictID)
 	if err != nil {
-		writeError(w, "error when calling /wallet/encrypt: "+err.Error(), http.StatusBadRequest)
+		writeError(w, "error when calling /wallet/init: "+err.Error(), http.StatusBadRequest)
 		return
 	}
-	writeJSON(w, WalletEncryptPOST{
+	writeJSON(w, WalletInitPOST{
 		PrimarySeed: seedStr,
 	})
 }
 
+// walletInitHandler handles API calls to /wallet/init.
+func (srv *Server) walletInitHandler(w http.ResponseWriter, req *http.Request) {
+	if req.Method == "POST" {
+		srv.walletInitHandlerPOST(w, req)
+		return
+	}
+	writeError(w, "unrecognized method when calling /wallet/init", http.StatusBadRequest)
+}
+
 // walletEncryptHandler handles API calls to /wallet/encrypt.
+// Replaced with /wallet/init kept for compatability
```

Comment:
first word of the docstring should be the function name.

Also, need to add a COMPAT message.

Something like this:

```
// walletEncryptHandler is a legacy alias for walletInitHandler.
//
// COMPAT v0.4.0